ui.apps/src/main/content/jcr_root/apps/aemtraining/components/content/newscard/clientlibs/newscard/js/jquery-3.6.0.min.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.org/license */
```

Comment:
Please don't add jQuery this way. It is just a foreign library. We don't add jar files to Git, do we ?) jQuery is no better))